src/Northwind.MusicStore.WebUI/Controllers/AccountController.cs

```diff
@@ -220,8 +220,12 @@ public ActionResult Manage(LocalPasswordViewModel viewModel)
                     {
                         WebSecurity.CreateAccount(User.Identity.Name, viewModel.NewPassword);
                         return RedirectToAction("Manage", new { Message = ManageMessageId.SetPasswordSuccess });
+                    }
+                    catch (MembershipCreateUserException e)
+                    {
+                        ModelState.AddModelError("", e);
                     }
-                    catch (Exception e)
```

Comment:
You've removed the catch all exception.
This is required so if an unexpected error occurs, an error is added to ModelState